ui/ui-tests/tests/institutions/detail.test.js

```diff
@@ -0,0 +1,43 @@
+const { routes, selectors } = require('../../utils/constants');
+const { createPollyInstance } = require('../../utils/polly');
+const { getMetaDescription } = require('../../utils/dom');
+const {
+  takeScreenShotForDesktop,
+  takeScreenShotForMobile,
+} = require('../../utils/screenshot');
+
+describe('Institution Detail', () => {
+  let polly;
+
+  beforeEach(async () => {
+    polly = await createPollyInstance('InstitutionDetail');
+    await page.goto(routes.public.institutionsDetail902624, {
+      waitUntil: 'networkidle0',
+    });
+  });
+
+  xit('should match image snapshot for an Institution', async () => {
+    await page.waitFor(selectors.searchResults);
+
+    const desktopSS = await takeScreenShotForDesktop(page);
+    expect(desktopSS).toMatchImageSnapshot();
+
+    const mobileSS = await takeScreenShotForMobile(page);
+    expect(mobileSS).toMatchImageSnapshot();
+  });
+
+  xit('sets institution name as document title', async () => {
```

Comment:
this can be enabled

Reply:
I didn't want to make the recordings and enable them yet because institutions is still under authentication on QA and I didn't want to add logins/logouts in the tests temporarily. And I didn't manage to do the recordings locally.